common/lib/xmodule/xmodule/tests/test_course_metadata_utils.py

```diff
@@ -159,6 +160,10 @@ def mock_strftime_localized(date_time, format_string):
                     advertised_start_unparsable.title()
                 ),
                 TestScenario(
+                    (test_datetime, advertised_start_bad_date, 'DATE_TIME', ugettext, mock_strftime_localized),
```

Comment:
Will the `mock_strftime_localized` return a `ValueError` for `advertised_start_bad_date`?  
To make sure, you can assert that `ValueError` is raised when called with `advertised_start_bad_date`.